README.md

```
@@ -102,10 +105,11 @@ The platform is designed to be brought up and down cleanly using the `invoke` ta
 1. `uv run invoke dev.deps` (Starts Infrahub, Temporal, Telemetry)
 2. `uv run invoke dev.lab-deploy` (Deploys Containerlab switches)
 3. `uv run invoke workers.start` (Starts your Python orchestration workers)
+4. `uv run invoke presentation.start` (Starts the self-service portal)
 
 ### 🔴 Shutting Down
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)